app/data/census_individual.json

```diff
@@ -2233,17 +2309,17 @@
                     "sections": [
                         {
                             "id": "employment-type-section",
-                            "number": "26",
+                            "number": "28",
                             "title": "{{ [answers.first_name, answers.last_name] | format_household_name }}",
                             "questions": [
                                 {
                                     "id": "employment-type-question",
                                     "title": "Last week were you:",
-                                    "number": "26",
+                                    "number": "28",
                                     "description": "Select all that apply",
                                     "guidance": [
                                         {
-                                            "description": "Include any paid work, including casual or temporary work, even if only for one hour"
+                                            "description": "<strong>Include</strong> any paid work, including casual or temporary work, even if only for one hour"
```

Comment:
From the google doc it looks like the whole thing should be bold. But there's inconsistency here across questions so, pinch of salt..